style.css

```diff
@@ -1 +1,180 @@
 /*  Add your styling here */
+* {
+  box-sizing: border-box;
+}
+
+main {
+  background-color: #ffc8dd;
+}
+
+/* bodygrid layout */
+body {
+  display: grid;
+  font-family: "Dancing Script", cursive;
+  grid:
+    "hero"
+    "body"
+    "footer";
+}
+
+header {
+  grid-area: hero;
+}
+
+main {
+  grid-area: body;
+}
+
+footer {
+  grid-area: footer;
+}
+
+/* headersection starts here */
+.nav-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  background-color: #cdb4db;
+  width: auto;
+  height: 300px;
+  /* border: 1px solid black; */
+}
+
+.nav-img {
+  width: 100%;
+  height: auto;
+}
+
+/* nav bar  */
+.nav-bar {
+  justify-content: center;
+}
+
+.nav-bar ul {
+  display: flex;
+}
+
+.nav-bar li {
+  list-style: none;
+}
+
+.nav-bar li a {
+  text-decoration: none;
+  padding: 10px;
+}
+
+.nav-bar li a:hover {
+  background-color: gray;
+}
+
+/* hamburger menu  */
+.toggle-button {
+  position: absolute;
+  top: 0.75rem;
+  right: 1rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 40px;
+  height: 30px;
+}
+
+.toggle-button .bar {
+  height: 3px;
+  width: 100%;
+  background-color: black;
+  border-radius: 10px;
+}
+
+@media only screen and (min-width: 540px) {
+  .nav-bar {
+    display: flex;
+    align-self: flex-end;
+  }
+}
+
+.nav-container p {
+  align-self: flex-end;
+  margin-right: 10px;
+  font-size: 15px;
+}
+
+/* main starts here */
+
+.hero-info-img {
+  display: flex;
+  flex-direction: column;
+}
+
+.heroImg {
+  width: 50%;
+  height: auto;
+  align-self: center;
+  margin: 20px 0;
+}
+
+.allImages {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, 226px);
+  grid-template-rows: 200px 200px;
+  grid-gap: 10px;
+  /* border: solid; */
+}
+
+.heroImages {
+  width: 100%;
+  height: 200px;
+}
+
+.hero-info-img h1 {
+  margin-left: 20px;
+  font-size: 50px;
+  color: #bde0fe;
```

Comment:
One way you can check out the contrast is by using Chrome Dev Tools and hover over elements (icon to click is top right in this screenshot, or command + shift + C on Mac):

<img width="528" alt="image" src="https://user-images.githubusercontent.com/77307047/185600790-dc3b8c4b-d86d-4fe8-8516-db48c1ec677c.png">

You can see that in this case the contrast between text and background is a little bit low - it would improve accessibility to have more contrast